shell/platform/darwin/ios/framework/Source/FlutterPlatformViews.mm

```diff
@@ -422,11 +441,25 @@ - (BOOL)flt_hasFirstResponderInViewHierarchySubtree {
       case kOpacity:
         embedded_view.alpha = (*iter)->GetAlphaFloat() * embedded_view.alpha;
         break;
-      case kBackdropFilter:
+      case kBackdropFilter: {
+        // We only support DlBlurImageFilter for BackdropFilter.
+        if (!(*iter)->GetFilter().asBlur())
+          continue;
+
+        // Sigma X is arbitrarily chosen as the radius value because Quartz only supports 1D
+        // rendering. DlBlurImageFilter's Tile Mode is not supported in CIGaussianBlurFilter so it
```

Comment:
nits: we didn't use `CIGaussianBlurFilter`, we used a CAFilter named gaussianBlur. how about "DlBlurImageFilter's Tile Mode is not supported in the gaussianBlur CAFilter in Quartz ..."